index.html

```diff
@@ -215,85 +241,6 @@
 
 			var originalZoomLevel = map.getZoom();
 
-			function setVisibilityState(params) {
```

Comment:
pruned some functions that we aren't using in this map.  We can always re-copy them from one of the other RYHT projects if we need them again.